app/assets/javascripts/Components/submission_file_manager.jsx

```diff
@@ -254,9 +256,52 @@ class SubmissionFileManager extends React.Component {
     );
   };
 
+  renderRequiredFiles = () => {
+    let requiredFilesBox;
+
+    if (this.state.requiredFiles.length > 0) {
+      requiredFilesBox = (
+        <div>
+          <p>{I18n.t("assignments.assignment_files")}</p>
+          {this.state.requiredFiles.map(filename => {
+            return (
+              <div key={filename}>
```

Comment:
I think the styling can be improved a bit here. Maybe use `p` tags to create more vertical space between each element. Also maybe use a bolded **(missing)** or something next to the files that still have to be submitted.